pytest_translations.py

```diff
@@ -169,6 +315,23 @@ def collect(self):
             self.name,
             self,
         )
+        if enchant:
+            parsed = _open_po_file(self.fspath)
+            lang_dict = parsed.metadata.get('Language', 'en')
```

Comment:
the naming could  be better. 

here `lang_dict` is the language string, 

leter on it's replaced with the `DictWithPWL`